CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ All notable changes to this project will be documented in this file based on the
 ### Improvements
 
 ### Deprecated
-
+ - Removed `Query\NumericRange`, use `Query\Range` instead [#1334](https://github.com/ruflin/Elastica/pull/1334)
```

Comment:
This probably goes more under BC Breaks as we didn't deprecated it but removed it.